.github/workflows/_buildpacks-release-package.yml

```diff
@@ -46,6 +46,8 @@ jobs:
 
       - name: Rust cache
         uses: Swatinem/rust-cache@v2
+        with:
+          key: ${{ inputs.buildpack_id }}-v${{ inputs.buildpack_version }}
```

Comment:
I don't think we want the version here, since:
- the version will change for each release, meaning the cache is never reused from one release to the next
- the version isn't what affects the contents of the build, things like the `Cargo.lock` do, and the Rust caching plugin already uses the SHA256 of that file

Just the ID on it's own should be good I think.